doc/external/pytorch_tutorials_hyperparameter_tuning_tutorial.py

```diff
@@ -0,0 +1,493 @@
+# -*- coding: utf-8 -*-
+# flake8: noqa
+"""
```

Comment:
we can definitely leave this as is, just talking out loud: can we somehow leverage these interspersed comments better? right now the only purpose of them is to make clear how this relates to the flow of the externally hosted tutorial (which is a good thing). but since the source file seems to be `rst` already, and we essentially put the whole content into this PR, why not go the whole way and render this on our docs, too? (e.g. by creating an `ipynb` notebook.) this would be duplicating content, of course. on the other hand, maybe it makes bi-directionally syncing such examples easier.

otherwise, maybe it's better just to indicate through comments in which section of the original article we currently are and remove the rest, since this file is intended for testing only, not to follow along

Reply:
interesting! ignore everything I said then.